src/Pagination/components/PageNumberButton.ts

```diff
@@ -1,22 +1,41 @@
-import { SFC, createElement } from "react";
+import { ReactElement, createElement } from "react";
 import * as classNames from "classnames";
```

Comment:
You can short this one.. not required tho

Reply:
Shortened it.